.github/workflows/check-no-external-libs.yml

```
@@ -43,4 +43,4 @@ jobs:
           for root, _, files in os.walk('src'):
               for file in files:
                   if file.endswith('.py'):
-                      check_imports(os.path.join(root, file))"
+                      check_imports(os.path.join(root, file))"
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)